README.md

```diff
@@ -1 +1,114 @@
 # OKRs 2018 Q4
+
+## Bring 'Code as Data' to market with enterprises
+
+### Ensure gitbase interoperability with 3rd party tools [DR]
+
+- [ ] Drivers (JS, PHP, Python, Ruby, Go, Java, .NET)
+- [ ] mysqldump
+- [ ] grafana
+- [ ] BI (Tableau, PowerBI, pick one)
+
+### Get gitbase enterprise ready [DR]
+
+- [ ] Authentication and authorization abstractions with basic MySQL auth implementation
```

Comment:
I cannot understand what is required here. Roles? read only users? ...

Reply:
@ajnavarro  We'll discuss during the design process, but the minimum implementation would be accepting authentication with user+pass from MySQL protocol, with a defined user+password in a file, and an authorizer that authorizes everything for any authenticated user.

However, the design should consider future integration with LDAP and Kerberos, as well as authorizing read/write operations independently.